.style.yapf

```diff
@@ -0,0 +1,5 @@
+[style]
```

Comment:
Is it possible to apply YAPF right now without changing every 2nd line in the code?

Reply:
I tried to find a good `yapf` config that didn't change everything. Can't find the best setup. The `cpplint` code does weird things with the indenting. Maybe we could circle round on that after this is merged?